.gitignore

```
@@ -1,4 +1,4 @@
-.DS_Store
+ .DS_Store
```

```suggestion
.DS_Store
```
Was adding the space intentional?

---

I think this is a merge issue